houseinput.json

```diff
@@ -10,33 +17,24 @@
     "windows": 2,
     "lights": 3,
     "doorsTo": [
+      "Entrance",
       "Garage",
-      "Living Room",
-      "Bedroom"
```

Comment:
We don't have a bedroom anymore?

Reply:
guess not :0.  People can sleep in the garage